include/materials/norsand.tcc

```diff
@@ -585,4 +482,4 @@ Eigen::Matrix<double, 6, 1> mpm::NorSand<Tdim>::compute_stress(
 
```

Comment:
Could we also summarized the computation of plastic strain inside the material utility?

Reply:
I think it is a good idea

Reply:
I think we can make a function to compute `pdstrain` (a scalar plastic deviatoric strain) in `material_utility` and also the `dpstrain` (a vector of plastic strain). But do you think the MohrCoulomb model can be consistent with the current definition?

Reply:
I used your way to compute the pdstrain before, and can get the good result. Please just use your method to compute the dstress first and then dpstrain and then pdstrain. I will use the general function in the MC and MCC model.

Reply:
Can we do this in another PR?

Reply:
I'm ok with that, but why another? I think it won't have too many changes.

Reply:
OK, sure.